README.md

```diff
@@ -47,10 +47,10 @@ very similar steps on most modern Linux distributions.
 
 If you run into dependency problems, look at `bistro/cmake/setup.cmake` for
 a full list of Bistro's external dependencies (direct and indirect).  We
-gratefully accept patches that improve Bistro's builds, or add support for
+gratefully accept patches that improve Bistro's builds or add support for
 various flavors of Linux and Mac OS.
 
-The binaries will be in `bistro/cmake/{Debug,Release}`.  Available build
+The binaries will be in `bistro/cmake/{Debug, Release}`.  Available build
```

Comment:
Let's not change this. The `{foo,bar}` syntax has clear significance to anybody who uses a Bourne-style shell. Adding whitespace here changes the meaning. This is not a good fix, since the `Release` directory does not, in fact, have a leading space.